### PR TITLE
Added SlimefunWarfare and Liquid

### PR DESCRIPTION
**Pull request by Seggan**
The title says it all

### CODE REVIEW

File: pages/Addons.md [hunk 1]
```diff
@@ -44,6 +44,8 @@ ExtraTools | Sfiguz | <details><summary>expand</summary>ExtraTools is a Slimefun
...
 Slimy Power Suits (WIP) | Seggan | <details><summary>expand</summary>This addon is an expansion for the popular addon LiteXpansion. It adds modular power suits to Slimefun.</details> | N/A | [GitHub](https://github.com/Seggan/Slimy-Power-Suits) | [Builds](https://thebusybiscuit.github.io/builds/Seggan/Slimy-Power-Suits/master/)
+SlimefunWarfare | Seggan | <details><summary>expand</summary>This addon adds guns, chemical grenades, and more war stuff to Slimefun</details> | N/A | [GitHub](https://github.com/Seggan/SlimefunWarfare) | [Builds](https://thebusybiscuit.github.io/builds/Seggan/SlimefunWarfare/master/)
+Liquid | Seggan, NCBPFluffyBear | <details><summary>expand</summary>This addon adds liquid version of Slimefun metals and alloys. It also adds ways to more effectiently make alloys with the molten metals</details> | N/A | [GitHub](https://github.com/Seggan/Liquid) | [Builds](https://thebusybiscuit.github.io/builds/Seggan/Liquid/master/)
```
Comment (TheBusyBiscuit) [inline pages/Addons.md:48]: Please add your addons to the bottom of the list and not somewhere in the middle...

Comment (TheBusyBiscuit) [inline pages/Addons.md:48]: `effectiently`... wut? You mean "efficiently" or "effecitvely".
Please avoid neologisms XD

**Review by TheBusyBiscuit — CHANGES_REQUESTED** [review]

File: pages/Addons.md (hunk 1 above)
Comment (Seggan) [reply pages/Addons.md:48]: well thats where the rest of my addons are...

Comment (Seggan) [reply pages/Addons.md:48]: oops

Comment (TheBusyBiscuit) [reply pages/Addons.md:48]: So?

**Review by TheBusyBiscuit — DISMISSED** [review]

**Review by svr333 — CHANGES_REQUESTED** [review]
For consistency purposes I would end the description with a `.`

File: pages/Addons.md (hunk 1 above)
Comment (svr333) [reply pages/Addons.md]: lol

**Review by svr333 — APPROVED** [review]
Looking good!

**Review by TheBusyBiscuit — APPROVED** [review]

**Pull request merged**